newIDE/app/src/ProjectManager/ProjectPropertiesDialog.js

```diff
@@ -213,127 +217,130 @@ function ProjectPropertiesDialog(props: Props) {
+                  key="hot-reload-preview-button"
+                  icon={<NewPreviewIcon />}
+                  label={<Trans>Run a preview (with loading screen)</Trans>}
+                  onClick={
+                    hotReloadPreviewButtonProps.launchProjectWithLoadingScreenPreview
+                  }
+                />
+              ) : null,
+            ]}
+            noTitleMargin
+            open={props.open}
+            onRequestClose={onCancelChanges}
+            fullHeight
+            flexBody
+            title={
+              <div>
+                <Tabs value={currentTab} onChange={setCurrentTab}>
+                  <Tab
+                    label={<Trans>Properties</Trans>}
+                    value={'properties'}
+                    key={'properties'}
+                  />
+                  <Tab
+                    label={<Trans>Loading Screen</Trans>}
+                    value={'loading-screen'}
+                    key={'loading-screen'}
+                  />
+                </Tabs>
+              </div>
+            }
+          >
+            {currentTab === 'properties' && (
+              <ColumnStackLayout expand noMargin>
+                <SemiControlledTextField
+                  floatingLabelText={<Trans>Game name</Trans>}
+                  fullWidth
+                  type="text"
+                  value={name}
+                  onChange={setName}
+                  autoFocus
+                />
+                <SemiControlledTextField
+                  floatingLabelText={<Trans>Version number (X.Y.Z)</Trans>}
+                  fullWidth
+                  hintText={defaultVersion}
+                  type="text"
+                  value={version}
+                  onChange={setVersion}
+                />
+                <SemiControlledTextField
+                  floatingLabelText={
+                    <Trans>Package name (for iOS and Android)</Trans>
+                  }
+                  fullWidth
+                  hintText={defaultPackageName}
+                  type="text"
+                  value={packageName}
+                  onChange={setPackageName}
+                  errorText={
+                    validatePackageName(packageName) ? (
+                      undefined
+                    ) : (
+                      <Trans>
+                        The package name is containing invalid characters or not
+                        following the convention "xxx.yyy.zzz" (numbers allowed
+                        after a letter only).
+                      </Trans>
+                    )
+                  }
+                />
+                <UsersAutocomplete
+                  userIds={project.getAuthorIds()}
+                  floatingLabelText={<Trans>Authors</Trans>}
+                  helperText={
+                    <Trans>
+                      Select the usernames of the authors of this project. They
+                      will be displayed in the order selected.
```

Comment:
`will be displayed in the selected order, if you publish this game as an example or in the community.`? 

(to stay fairly generic).